docs/io/visualization/convergence_plot.ipynb

```diff
@@ -0,0 +1,846 @@
+{
```

Comment:
<p>This is excellent, as well as description of changing colors section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Thank you!!